backend/modules/model_gateway/__init__.py

```diff
@@ -1 +1,19 @@
+# When using locally, you can create your own models_config.json file before running docker-compose.
+# The following code is used for tf-deployments
+import json
+import os
 
+from backend.logger import logger
+
+# Check if models_config.sample.json exists but not models_config.json
+# If models_config.sample.json exists, copy it to models_config.json
+if os.path.exists("./models_config.sample.json") and not os.path.exists(
+    "./models_config.json"
+):
```

Comment:
This can break depending on what is the cwd, the path should be made relative to this file using os.path operations on `__file__`

Reply:
changed.